src/app/Shared/MatchExpression/MatchExpressionVisualizer.tsx

```diff
@@ -98,6 +98,11 @@ export const MatchExpressionVisualizer: React.FC<MatchExpressionVisualizerProps>
       <StackItem>
         <LayoutRadioGroup onChange={setIsGraph} />
       </StackItem>
+      <StackItem>
+        <Title headingLevel="h2" size="lg" className="breadcrumb-styled-title">
```

Comment:
You probably want to add this title outside the `MatchExpressionVisualizer` component. 

This way, u can easily control whether to show the title or not. This helps not to add a title when in credential modal because there is already one.

![image](https://github.com/cryostatio/cryostat-web/assets/68053619/0bcf90e2-74c7-4307-9d17-bdcd3a8b421b)

Reply:
@aali309